<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashWindow</class>
 <widget class="QDialog" name="SplashWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tStackedWidget" name="stackedWidget">
     <widget class="SplashWidget" name="splashPage"/>
     <widget class="CrashWidget" name="crashPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SplashWidget</class>
   <extends>QWidget</extends>
   <header>splash/splashwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrashWidget</class>
   <extends>QWidget</extends>
   <header>crash/crashwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
